src/util/datablock/datablock.h

```diff
@@ -81,3 +81,7 @@ void DataBlock_DeleteItem(DataBlock *dataBlock, uint64_t idx);
 
 // Free block.
 void DataBlock_Free(DataBlock *block);
+
+/*--------- DataBlock Item API ---------*/
+
+DataBlockItemHeader *DataBlock_GetItemHeader(const DataBlock *dataBlock, uint64_t idx);
```

Comment:
`void DataBlock_Free(DataBlock *block);`
Should be the last function in this file,

Also please add documentation for `DataBlock_GetItemHeader`